core/state_transition.go

```diff
@@ -144,28 +144,28 @@ type Message struct {
 	// This field will be set to true for operations like RPC eth_call.
 	SkipAccountChecks bool
 
-	IsSystemTx    bool                // IsSystemTx indicates the message, if also a deposit, does not emit gas usage.
-	IsDepositTx   bool                // IsDepositTx indicates the message is force-included and can persist a mint.
-	Mint          *big.Int            // Mint is the amount to mint before EVM processing, or nil if there is no minting.
-	RollupDataGas types.RollupGasData // RollupDataGas indicates the rollup cost of the message, 0 if not a rollup or no cost.
+	IsSystemTx  bool             // IsSystemTx indicates the message, if also a deposit, does not emit gas usage.
+	IsDepositTx bool             // IsDepositTx indicates the message is force-included and can persist a mint.
+	Mint        *big.Int         // Mint is the amount to mint before EVM processing, or nil if there is no minting.
+	L1CostData  types.L1CostData // L1CostData caches data to compute the fee we charge for L1 data availability
```

Comment:
Why exactly was this renamed? Just because this same function will charge for DA in the world of alt da

Reply:
mostly in preparation of adding fancier compression estimation, which will require we cache data other than zero/one counting.

Reply:
I still don't follow - this function isn't necessarily tied to L1 just a DA

Reply:
Oh I see what you are saying. I wasn't trying to be DA agnostic, I chose the name because the value is (currently) only used by the L1CostFunc.   Would you prefer "DACostData"?

Reply:
aligned on `RollupCostData` in a meeting

Reply:
done!